src/utils/daos_hdlr.h

```diff
@@ -190,6 +191,14 @@ int pool_autotest_hdlr(struct cmd_args_s *ap);
  * int pool_stat_hdlr(struct cmd_args_s *ap);
  */
 
+/* general datamover operations */
+void dm_cont_free_usr_attrs(int n, char ***_names, void ***_buffers, size_t **_sizes);
+int dm_cont_get_usr_attrs(struct cmd_args_s *ap, daos_handle_t coh, int *_n, char ***_names,
+			  void ***_buffers, size_t **_sizes);
+int dm_cont_get_all_props(struct cmd_args_s *ap, daos_handle_t coh, daos_prop_t **_props,
+			  bool get_oid, bool get_label, bool get_roots);
+int dm_copy_usr_attrs(struct cmd_args_s *ap, daos_handle_t src_coh, daos_handle_t dst_coh);
+
```

Comment:
@daltonbohning these are wrappers that look up the symbol in the library, you cannot call them directly.

Reply:
@dsikich But they are being defined in `daos-hdlr.c` as well?
https://github.com/daos-stack/daos/blob/c357bbb505f43fcbb18114e8cf407dae236ba59d/src/utils/daos_hdlr.c#L2502-L2517

Reply:
@daltonbohning these are not actually defined in daos_serialize.c because they do not require HDF5 so I was trying to limit how many functions I had to lookup in the library with dlsym. I think they could either be "wrappers" where they are defined in daos_serialize.c then I would have to have to look them up, or I just defined them directly in daos_hdlr.c.

Reply:
@daltonbohning I also remember now I wanted to use some of these functions in the case that serialization was not being used. For instance to copy properties when data is being transferred from DAOS->DAOS.

Reply:
Oh, I see... For some reason I thought they had been defined in daos_serialize.c. Please disregard 👍